src/ethereum_test/types.py

```diff
@@ -181,19 +186,69 @@ def must_contain(self, other: "Storage"):
                     )
                 )
 
+    def must_be_equal(self, other: "Storage"):
+        """
+        Succeeds only if "self" is equal to "other" storage.
+        """
+        # Test keys contained in both storage objects
+        for k in self.data.keys() & other.data.keys():
+            if self.data[k] != other.data[k]:
+                raise Exception(
+                    "incorrect value for key {0}: {1}!={2}".format(
+                        Storage.key_value_to_string(k),
+                        Storage.key_value_to_string(self.data[k]),
+                        Storage.key_value_to_string(other.data[k]),
+                    )
+                )
+        # Test keys contained in either one of the storage objects
+        for k in self.data.keys() ^ other.data.keys():
+            if k in self.data:
+                if self.data[k] != 0:
+                    raise Exception(
+                        "expected key {0}={1} not found in storage".format(
+                            Storage.key_value_to_string(k),
+                            Storage.key_value_to_string(self.data[k]),
+                        )
+                    )
+            elif other.data[k] != 0:
+                raise Exception(
+                    "unexpected key {0}={1} found in storage".format(
+                        Storage.key_value_to_string(k),
+                        Storage.key_value_to_string(other.data[k]),
+                    )
+                )
+
 
 @dataclass(kw_only=True)
 class Account:
     """
     State associated with an address.
     """
 
-    nonce: Optional[int] = None
-    balance: Optional[int] = None
-    code: Optional[Union[bytes, str, Code]] = None
-    storage: Optional[
-        Union[Storage, Dict[Union[str, int], Union[str, int]]]
-    ] = None
+    nonce: int | None = None
+    """
```

Comment:
Are python comments really below the actual code definition?

Reply:
Yes! check it out: https://peps.python.org/pep-0224/